test/compression.js

```diff
@@ -676,6 +676,35 @@ describe('compression()', function () {
       .end()
     })
   })
+
+  describe('when callbacks are used', function () {
+    it('should call the passed callbacks', function(done){
+      var callbacks = 0;
+      var server = createServer({ threshold: '1kb' }, function (req, res) {
```

Comment:
Typically you should only pass in the options that are relevant to the test. Here you are passing in `threshold: '1kb'`, which doesn't seem to be necessary for the test (and is even the default setting). Can this be removed? If not, what is the purpose for specifying it explicitly?